packages/web/integrations/nomic/transfer.tsx

```diff
@@ -181,6 +181,18 @@ const NomicTransfer: FunctionComponent<
         return;
       }
 
+      if (withdrawAmountConfig.sendCurrency.coinDenom !== "usat") {
```

Comment:
What are your thoughts on disabling the submit button instead of waiting for the user to click withdraw